users/rwijtvliet/config.h

```diff
@@ -0,0 +1,31 @@
+
+
```

Comment:
This needs a license header, and it needs:
```suggestion
#pragma once
```